### PR TITLE
avatar component is done

### PR DESCRIPTION
**Pull request by JantuDeb**
image
image

### CODE REVIEW

**Comment by JantuDeb** [comment]
[This commit contains all the css for avatar component](/JantuDeb/FlamingoUI/pull/3/commits/039a46054d806bf37883adcd6f56e3b2cfeef227)

**Review by rkapoor10 — COMMENTED** [review]
Given some ideas. May come in handy. Great going. 👍

File: components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  border-radius: 50%;
+}
+
+.avatar-large {
+  vertical-align: middle;
+  width: 8rem;
+  height: 8rem;
+  border-radius: 50%;
+}
+
```
[the remark this answers is not in the document]
Comment (JantuDeb) [reply components/avatar.css:28]: Thanks, will include these changes.

**Comment by harshitadaswani** [comment]
just include the above-mentioned changes and you are good to go!

**Pull request merged**